components/src/core/list-item-tag/list-item-tag.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="pxb-list-item-tag-wrapper" 
+    
+    [style.backgroundColor]="backgroundColor">
+    
+    <div class="mat-small" [class.pxb-list-item-tag-label]="label" [style.color]="fontColor">
+        {{ label.toUpperCase() }}
```

Comment:
How does a user set the style of the typography they want to use? (What's the equivalent to the `variant` prop from the react component?)

Reply:
react uses `variant=overline` by default. And unfortunately, Angular's obsolete typography does not have an overline class in their typography library, which also changes the capitalization with just CSS, and what you are seeing right here is apparently a custom style. 

So, I guess, at least use CSS property `text-transform: uppercase;` for this one